pkg/e2e/state/prometheus.go

```diff
@@ -26,6 +31,27 @@ var _ = ginkgo.Describe(clusterStateInformingName, func() {
 		// common for a running prometheus instance
 		cmd := promCollectCmd + " > " + runner.DefaultRunner.OutputDir + "/prometheus.tar.gz; err=$? ; if (( $err != 0 )) ; then exit $err ; fi"
 
+		poderr := wait.PollImmediate(2*time.Second, 5*time.Minute, func() (bool, error) {
+			podCount := 0
+			var podErr error
+			list, _ := verify.FilterPods("openshift-monitoring", "app=prometheus", h)
```

Comment:
Why discard the error here?

Reply:
The FilterPods function prints an error so I thought that would be enough but I guess including the error here is far more robust.